pkg/cloud/cloud.go

```diff
@@ -500,7 +496,7 @@ func (c *cloud) ResizeOrModifyDisk(ctx context.Context, volumeID string, newSize
 	if options.VolumeType != "" {
 		req.VolumeType = aws.String(options.VolumeType)
 	}
-	if options.VolumeType == VolumeTypeGP3 {
+	if options.Throughput != 0 {
```

Comment:
What happens if you attempt to modify throughput on volume types that don't support it?

Reply:
AWS API returns an error -> we return that to the caller